src/ui/customBreakpointsUI.ts

```diff
@@ -206,4 +295,45 @@ export function registerCustomBreakpointsUI(
       provider.setEnabled(provider.allBreakpoints.map(bp => [bp, false]));
     }),
   );
+
+  context.subscriptions.push(
+    vscode.commands.registerCommand(Commands.AddXHRBreakpoints, () => {
+      const inputBox = vscode.window.createInputBox();
+      inputBox.title = 'Add XHR Breakpoint';
+      inputBox.placeholder = 'Enter a URL or a pattern to match';
```

Comment:
I'm not sure it's clear to the user what pattern means here. Is it a regex, a glob match, something else? Maybe we copy devtools' wording:

<img width="377" alt="image" src="https://github.com/microsoft/vscode-js-debug/assets/2230985/356f5a59-6da2-4d4c-8924-fe06485cbb0f">